### DISCUSSION TITLE
Storing a set of annotations

### DISCUSSION THREAD

**Discussion by dummifiedme** [pymupdf/PyMuPDF#1575] [Q&A]
I intend to store some `Annot` objects in a variable. Is it possible?
I intend to store all the rect type annotations in one variable (matrix, dict or list (whatever it is called o_O)) and then store all the annotations that lies within this rect into another variable, to make sure they are printed in the output together...

PS. I am not very well versed with programming :/ Trying to learn.

**Comment by JorjMcKie** [comment]
yes, that's possible

**Reply by dummifiedme** [reply]
How to? Can you please give an example?

**Comment by JorjMcKie** [comment]
Look at my snippet in #1571, which associates annotations with the list of text blocks.
